src/backend/utils/misc/etc/hawq-site.xml

```diff
@@ -20,103 +20,133 @@ under the License.
-    <property>
-        <name>hawq_rm_yarn_address</name>
-        <value>localhost:8032</value>
-        <description>The address of YARN resource manager server.</description>
-    </property>
-
-    <property>
-        <name>hawq_rm_yarn_scheduler_address</name>
-        <value>localhost:8030</value>
-        <description>The address of YARN scheduler server.</description>
-    </property>
-
-    <property>
-        <name>hawq_rm_yarn_queue_name</name>
-        <value>default</value>
-        <description>The YARN queue name to register hawq resource manager.</description>
-    </property>
-
-    <property>
-        <name>hawq_rm_yarn_app_name</name>
-        <value>hawq</value>
-        <description>The application name to register hawq resource manager in YARN.</description>
-    </property>
-
+	<!-- HAWQ resource manager parameters -->
+	<property>
+		<name>hawq_global_rm_type</name>
+		<value>none</value>
+		<description>The resource manager type to start for allocating resource.
+					 'none' means hawq resource manager exclusively uses whole
+					 cluster; 'yarn' means hawq resource manager contacts YARN
+					 resource manager to negotiate resource.
+		</description>
+	</property>
+
+	<property>
+		<name>hawq_rm_memory_limit_perseg</name>
+		<value>64GB</value>
+		<description>The limit of memory usage in a hawq segment when
+					 hawq_global_rm_type is set 'none'.
+		</description>
+	</property>
+
+	<property>
+		<name>hawq_rm_nvcore_limit_perseg</name>
+		<value>16</value>
+		<description>The limit of virtual core usage in a hawq segment when
+					 hawq_global_rm_type is set 'none'.
+		</description>
+	</property>
+
+	<property>
+		<name>hawq_rm_yarn_address</name>
+		<value>localhost:8032</value>
+		<description>The address of YARN resource manager server.</description>
+	</property>
+
+	<property>
+		<name>hawq_rm_yarn_scheduler_address</name>
+		<value>localhost:8030</value>
+		<description>The address of YARN scheduler server.</description>
+	</property>
+
+	<property>
+		<name>hawq_rm_yarn_queue_name</name>
+		<value>default</value>
+		<description>The YARN queue name to register hawq resource manager.</description>
+	</property>
+
+	<property>
+		<name>hawq_rm_yarn_app_name</name>
+		<value>hawq</value>
+		<description>The application name to register hawq resource manager in YARN.</description>
+	</property>
+	<!-- HAWQ resource manager parameters end here. -->
+
+	<!-- HAWQ resource enforcement parameters -->
+	<property>
+		<name>hawq_re_cpu_enable</name>
+		<value>false</value>
+		<description>The control to enable/disable CPU resource enforcement.</description>
+	</property>
+
+	<property>
+		<name>hawq_re_cgroup_mount_point</name>
+		<value>/sys/fs/cgroup</value>
+		<description>The mount point of CGroup file system for resource enforcement.
+					 For example, /sys/fs/cgroup/cpu/hawq for CPU sub-system.
+		</description>
+	</property>
+
+	<property>
+		<name>hawq_re_cgroup_hierarchy_name</name>
+		<value>hawq</value>
```

Comment:
Ambari currently sets this value as hadoop-yarn: https://github.com/Pivotal-Hadoop/ambari/blob/trunk/ambari-server/src/main/resources/common-services/HAWQ/2.0.0/configuration/hawq-site.xml#L169

Is this a problem ? Which value should be used ?